Dockerfile.cuda11.8

```diff
@@ -0,0 +1,43 @@
+FROM nvidia/cuda:11.8.0-runtime-ubuntu22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get install -y software-properties-common
+RUN add-apt-repository -y ppa:deadsnakes/ppa
+RUN apt-get update
+RUN apt-get install -y python3.11 python3.11-venv python3.11-dev python3-pip git cuda-nvcc-11-8 cuda-toolkit-11-8 vim
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.11 1
+RUN python3.11 -m ensurepip --upgrade
+RUN python3.11 -m pip install --upgrade pip setuptools wheel
+RUN apt-get install -y --no-install-recommends pandoc
+RUN apt-get -y clean
+RUN rm -rf /var/lib/apt/lists/*
+
+# Define volumes
+VOLUME /app/model_files
+VOLUME /app/vector_db_data
+
+# Set environment variables
+ENV VECTORDB_DIR=/app/vector_db_data
+ENV SENTENCE_TRANSFORMERS_HOME=/app/model_files
+ENV TRANSFORMERS_CACHE=/app/model_files
+ENV DEBIAN_FRONTEND=dialog
+
+# Set working directory
+WORKDIR /app
+
+# Copy requirements files
+COPY requirements.cuda.txt .
+
+# Install requirements
+RUN python3.11 -m pip install --no-cache-dir --no-deps -r requirements.cuda.txt
+RUN CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3 -m pip install llama-cpp-python --force-reinstall
+RUN python3 -m pip install torch --force-reinstall --no-cache-dir
```

Comment:
not sure if only cublas is installed here, maybe it would help to also install openblas.

we should get rid of `--force-reinstall` for `llama-cpp-python` by removing the dep from the requirements file. The same goes for `torch`. The cuda things do not work for torch because we use a cpu specific version in the reqs file.  
It should be enough to do `pip freeze > requirements.cuda.txt` and replace the `unstructured` dependency to use the git version as it is right now.